buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/CopyRestTestsTask.java

```diff
@@ -140,9 +141,12 @@ public FileTree getInputDir() {
     @OutputDirectory
     public File getOutputDir() {
         return new File(
-            getSourceSet().orElseThrow(() -> new IllegalArgumentException("could not find source set [" + sourceSetName + "]"))
-                .getOutput()
-                .getResourcesDir(),
+            new File(
```

Comment:
Similar to my comment in the transform task, can we refactor this such that the task simeply has `setOutputDir(File outputDir)` which is provided by the plugin?

Reply:
yeah ... pretty sure this can be handled by the plugins that wire this in this task...can't seem to remember the exact reason I went with this approach but I'll take another look. 

However, I would prefer in a follow up PR (which I should be able to do in the next day or two). In the (short) meanwhile I changed the variable name to make it slightly less confusing.